bundle/manifests/rhoai-operator-metrics-reader_rbac.authorization.k8s.io_v1_clusterrole.yaml

```diff
@@ -2,15 +2,9 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: prometheus-k8s-viewer
+  name: rhoai-operator-metrics-reader
```

Comment:
Any reason this clusterrole is updated?

Reply:
I didn't update this manually. It updated by deployment name change after running `make bundle` cmd.